sklearn/calibration.py

```diff
@@ -453,9 +460,16 @@ def fit(self, X, y, sample_weight=None, **fit_params):
                     n_jobs=self.n_jobs,
                     params=routed_params.estimator.fit,
                 )
-                predictions = _compute_predictions(
-                    pred_method, method_name, X, n_classes
-                )
+                # Reshape in the binary case
+                if len(self.classes_) == 2:
```

Comment:
```suggestion
                if len(self.classes_) == 2:
                    # reshape from (n_samples,) to (n_samples, 1) for binary case
```

Reply:
Hmm actually maybe I should use `predictions.ndim` here for consistency.